sources/packages/forms/src/form-definitions/partnerincomeappeal.json

```diff
@@ -0,0 +1,407 @@
+{
+    "title": "Partner Income Appeal",
+    "name": "partnerIncomeAppeal",
```

Comment:
As a suggestion, can we start all appeal forms with some suffix like _appeal_ to have them all grouped-ish by name? For instance:
```
"title": "Appeals - Partner Income",
"name": "appealsPartnerIncome",
"path": "appealspartnerincome"
```

Reply:
Currently all appeal forms(part of this PR) are suffixed with appeal. You meant prefix?

Reply:
Yes, right, prefix as per the example posted.